scan_topic : "/scan"
scan_queue : 10
point_cloud_topic : "/rslidar_points"
point_cloud_queue : 10
vel_topic : "/cmd_vel"
vel_queue : 10
vis_topic : "/visualization_marker"
vis_queue : 10
imu_topic : "/imu0"
imu_queue : 10
start_stop_srv : "start_stop_server"
kp_linear : 1
kp_angular : 0.33
distance_limit : 2
angular_deadband : 0.2
accel_change : 4